magento/ece-tools
# MCLOUD-6658: Enable 2 level cache by default

### Description

Since ECE-Tools release 2002.1.1, we can enable L2 cache with variables: https://devdocs.magento.com/cloud/env/variables-deploy.html#redis_backend

### Internal task

1. https://jira.corp.magento.com/browse/MCLOUD-6658

### Manual testing scenarios

- run deployment for Magento version 2.3.5 and higher (without custom cache configurations)
- check that after deployment L2 was enabled by default

### Release notes

For user-facing changes, add a meaningful release note. For examples, see [Magento Cloud ECE-tools release notes](https://devdocs.magento.com/cloud/release-notes/ece-release-notes.html).

### Associated documentation updates

Add link to Magento DevDocs PR or Issue, if needed.

### Contribution checklist
